packages/cli/src/commands/translate/sync.ts

```diff
@@ -28,23 +36,46 @@ const sync: CommandModule<{ path?: string }, { path?: string }> = {
       } satisfies Partial<TranslationOptions>;
 
       if (isPhrasesBuiltInLanguageTag(languageTag)) {
-        void syncTranslation({
+        await syncTranslation({
           ...baseOptions,
           packageName: 'phrases',
           languageTag,
         });
       }
 
       if (isPhrasesUiBuiltInLanguageTag(languageTag)) {
-        void syncTranslation({
+        await syncTranslation({
           ...baseOptions,
           packageName: 'phrases-ui',
           languageTag,
         });
       }
     }
+    /* eslint-enable no-await-in-loop */
 
     await queue.onIdle();
+
+    const spinner = ora({
+      text: 'Running `eslint --fix` for locales',
+    }).start();
+
+    const phrasesPath = path.join(instancePath, 'packages/phrases');
+    const localesPath = path.join(phrasesPath, 'src/locales');
+    await execPromise(
+      'pnpm',
+      ['eslint', '--ext', '.ts', path.relative(phrasesPath, localesPath), '--fix'],
+      { cwd: phrasesPath }
+    );
+
+    const phrasesUiPath = path.join(instancePath, 'packages/phrases-ui');
+    const localesUiPath = path.join(phrasesUiPath, 'src/locales');
+    await execPromise(
+      'pnpm',
+      ['eslint', '--ext', '.ts', path.relative(phrasesUiPath, localesUiPath), '--fix'],
+      { cwd: phrasesUiPath }
+    );
+
+    spinner.succeed('Ran `eslint --fix` for locales');
```

Comment:
should we create a util instead of copy paste?

Reply:
Yeah, I was also not quite satisfied with this part. Will create a util method instead

Reply:
Updated @gao-sun